socorro/external/es/supersearch.py

```diff
@@ -428,7 +428,7 @@ def get(self, **kwargs):
                 total = search.count()
 
                 aggregations = getattr(results, "aggregations", {})
-                if aggregations:
+                if isinstance(aggregations, AggResponse):
                     aggregations = self.format_aggregations(aggregations)
```

Comment:
With ES 8, the results either:

1. does not have an `aggregations` attribute at all in which case the `aggregations` variable points to a `{}` value
2. does have an `aggregations` attribute which points to a `AggResponse` object in which case, that gets run through `format_aggregations()` which returns a dict

That seems good to me.

`aggregations` points to a value with two very different types. We're not doing type checking, so it's just mildly not great. If we were to switch to type checking, then this would be more obviously not great.